src/DotNetWorker/Hosting/FunctionsWorkerHost.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.Extensions.Hosting
+{
+    /// <summary>
+    /// Hosing helpers to work with Azure Functiosn Workers
+    /// </summary>
+    public static class FunctionsWorkerHost
```

Comment:
This is primarily for some scenarios that won't impact some of the mainline samples, but will allow for quick configuration and execution of the host with reduced ceremony:
``` csharp
FunctionsWorkerHost.CreateDefaultAndRunAsync();
```

Reply:
I think I actually like 
```csharp
FunctionsWorkerHost.RunDefaultAsync();
```

Reply:
The use of `Host` here seems confusing since there's already a concept of a functions `Host`. Can we drop the `Host`? Or use `Application`?

Reply:
We could, but was afraid that would be confuses with `FunctionApplication`.... I'm also not a huge fan, but `Host` would be consistent with the terminology used in ASP.NET, Generic Host and others